app/controllers/host_shelters/properties_controller.rb

```diff
@@ -0,0 +1,24 @@
+class HostShelters::PropertiesController < HostShelters::HostSheltersController
+  before_action :auth_user_check?
+
+  def new
+    @property = Property.new
+  end
+
+  def create
+  property = Property.create(safe_params)
+   binding.pry
+   if property.save
+     session[:property_id] = property.id
+     redirect_to root_path
+   else
+     render file: 404
```

Comment:
I know this isn't a big deal right now, but this line should be `render file: 'public/404.html'`

Reply:
I was reading up and people say to use "ActionController::RoutingError.new('Not Found')" since it's more flexible. Maybe something to look into?
https://stackoverflow.com/questions/2385799/how-to-redirect-to-a-404-in-rails